azkaban-common/src/test/java/azkaban/executor/ExecutionFlowDaoTest.java

```diff
@@ -268,9 +268,10 @@ public void testFetchStaleFlows() throws IOException, ExecutorManagerException {
     // Flow executions in PREPARING state
     final long lessThan15Minutes = System.currentTimeMillis() - TimeUnit.MINUTES.toMillis(5);
     final ExecutableFlow flow1 = submitNewFlow("exectest1", "exec1", lessThan15Minutes,
-        ExecutionOptions.DEFAULT_FLOW_PRIORITY, Status.PREPARING, Optional.empty(), DispatchMethod.POLL);
```

Comment:
hmm why do you change from POLL to Containerized?

Reply:
so as this `flow1` can be queried, non Containerized flow won't be returned